ClocalReg.js

```diff
@@ -0,0 +1,108 @@
+const fs = require('fs');
+const path = require('path');
+const files = require('./lib/files/index.js');
+const inquirer  = require('./lib/inquirer');
+const ConfigstoreBase = require('configstore');
+const conf = new ConfigstoreBase('ginit');
+const octokit = require('@octokit/rest')();
+const Configstore = require('configstore');
+const API_Switch = require('../package.json');
+const assert = require('lodash');

```

Comment:
Do we have *Lodash* in the `package.json` file? The next lines are also coming from *'nowhere'*.